### DISCUSSION TITLE
PIM Installation

### DISCUSSION THREAD

**Discussion by ahlemgrissa** [atrocore/atrocore#186] [General]
i want to install PIM akeneo and i have many problem

**Comment by atrolex** [comment]
Please submit details and screenshots, if you mean AtroPIM. If you really want to install Akeneo, we cannot help you :)

**Comment by rdtandel** [comment]
![image](https://user-images.githubusercontent.com/14922373/145848584-ebb6d297-ae52-4287-8a8a-6d13ee4ce046.png)
i also having same issue

**Reply by rratsun** [reply]
You should use composer.phar which is provided with the project. It looks like you use your own composer.

```php composer.phar update```
